src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -42,37 +42,31 @@ MOI.delete(model::MOI.ModelLike, c::NormInfinityBridge) = MOI.delete(model, c.nn
 
 # Attributes, Bridge acting as a constraint
 function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintFunction, c::NormInfinityBridge{T, F}) where {T, F}
-    ge = MOIU.eachscalar(MOI.get(model, MOI.ConstraintFunction(), c.nn_index))
-    t = MOIU.operate!(/, T, sum(ge), T(length(ge)))
-    d = div(length(ge), 2)
-    x = MOIU.operate!(-, T, ge[(d + 1):end], ge[1:d])
-    # x = x / T(2)
-    # x = MOIU.operate!(/, T, x, T(2))
-    # for i in 1:d
-    #     MOIU.operate_output_index!(/, T, i, x, T(2))
-    # end
-    @show MOIU.canonicalize!(MOIU.operate(vcat, T, t, x))
+    nn_func = MOIU.eachscalar(MOI.get(model, MOI.ConstraintFunction(), c.nn_index))
+    t = MOIU.operate!(/, T, sum(nn_func), T(length(nn_func)))
+    d = div(length(nn_func), 2)
+    x = MOIU.operate!(/, T, MOIU.operate!(-, T, nn_func[(d + 1):end], nn_func[1:d]), T(2))
```

Comment:
@blegat This line fails because I need to divide a VAF by 2. If you could add the necessary method for `operate` in this PR (or in master and I will merge in) that would be a great help.

Reply:
Why are the tests passing if the line fails ?

Reply:
my comment was 2 days ago and I've merged in changes from master. works now.